script/tool/lib/src/publish_plugin_command.dart

```diff
@@ -64,13 +68,31 @@ class PublishPluginCommand extends PluginCommand {
       // Flutter convention is to use "upstream" for the single source of truth, and "origin" for personal forks.
       defaultsTo: 'upstream',
     );
+    argParser.addFlag(
+      _allFlag,
```

Comment:
Maybe we should call this flag `all-changed`? `--all` sounds more like it should do something like the Melos publish-all command.